src/rpdk/core/test.py

```diff
@@ -52,14 +54,61 @@ def temporary_ini_file():
         yield str(path)
 
 
-def get_overrides(root):
+def get_cloudformation_exports(region_name, endpoint_url):
+    session = create_sdk_session(region_name)
+    cfn_client = session.client("cloudformation", endpoint_url=endpoint_url)
+    next_token = ""
+    exports = {}
+    while next_token is not None:
```

Comment:
[Paginators](https://boto3.amazonaws.com/v1/documentation/api/latest/guide/paginators.html) might be useful here and simplify the logic instead of using the next token approach.

https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/cloudformation.html#CloudFormation.Paginator.ListExports

Reply:
Thanks for the suggestion, will use that instead